checkov/terraform/runner.py

```diff
@@ -241,12 +243,18 @@ def get_graph_checks_report(self, root_folder: str, runner_filter: RunnerFilter)
                         referrer_id = self._find_id_for_referrer(tf_path)
                         if referrer_id:
                             resource = f'{referrer_id}.{resource_id}'
+                    entity_config = self.get_graph_resource_entity_config(entity, entity_context)
+                    censored_code_lines = entity_context.get('code_lines') if not resource_attributes_to_omit else \
+                        omit_secret_value_from_graph_checks(check=check, check_result=check_result,
```

Comment:
@arielkru This means we will need to implement it for every framework (i.e. CFN, K8s etc)?
Is there a common solution?

Reply:
@nimrodkor simplified this to `censored_code_lines = omit_secret_value_from_graph_checks(..)` that can be implemented in any framework if needed. Currently we don't have neither the `resource_attributes_to_omit` set for any other framework or secret category graph based checks, so I think it's not relevant in the rest (right?). This is implemented in terraform runner but is actually just for terraform plan graph checks.